Jenkinsfile

```diff
@@ -130,7 +134,9 @@ pipeline {
                       cleanDir("${WORKSPACE}/${BASE_DIR}")
```

Comment:
if something happened here the error would not be marked inside the correct stage, isn't it?

Reply:
You are right, I'll ensure it is handled correctly